### PR TITLE
Enable the `..` argument for git log

### PR DESCRIPTION
**Pull request by alexr00**
This will return the commits that the `toRef` has but the `fromRef` does not.

### CODE REVIEW

File: extensions/git/src/git.ts [hunk 1]
```diff
@@ -1017,7 +1017,14 @@ export class Repository {
 
 	async log(options?: LogOptions): Promise<Commit[]> {
 		const maxEntries = options?.maxEntries ?? 32;
-		const args = ['log', `-n${maxEntries}`, `--format=${COMMIT_FORMAT}`, '-z', '--'];
+		const args = ['log', `-n${maxEntries}`, `--format=${COMMIT_FORMAT}`, '-z'];
```
Comment (rperez030) [inline extensions/git/src/git.ts:1020]: Please ignore my comment.

**Comment by lszomoru** [comment]
Thank you very much for submitting the pull request. These changes address the `..` special notation but will not work with the `...` notation. I think that we should do something that is more generic to make sure that both notations are handled. Maybe something like this:
```ts
readonly range?: string;
```
```ts
if (options?.range) {
	args.push(options.range);
}
```

**Review by lszomoru — CHANGES_REQUESTED** [review]
Please feel free to ping me if you would like to discuss this more. Thanks!

**Comment by alexr00** [comment]
Using a `range` instead makes sense to me. I have update the PR with this change.

**Comment by lszomoru** [comment]
@alexr00, is this something that you would like to get checked in in August or can this slip to September?

**Review by lszomoru — APPROVED** [review]

**Pull request merged**